x/accesscontrol/keeper/keeper.go

```diff
@@ -226,6 +226,25 @@ func BuildSelectorOps(accessOps []acltypes.AccessOperationWithSelector, senderBe
 				opWithSelector.Operation.IdentifierTemplate,
 				hex.EncodeToString(lengthPrefixed),
 			)
+		case acltypes.AccessOperationSelectorType_CONTRACT_ADDRESS:
+			contractAddress, err := sdk.AccAddressFromBech32(opWithSelector.Selector)
+			if err != nil {
+				return []acltypes.AccessOperationWithSelector{}, err
+			}
+			opWithSelector.Operation.IdentifierTemplate = fmt.Sprintf(
+				opWithSelector.Operation.IdentifierTemplate,
+				hex.EncodeToString(contractAddress),
+			)
+		case acltypes.AccessOperationSelectorType_JQ_MESSAGE_CONDITIONAL:
```

Comment:
out of curiosity, what is an example of a jq use case?

Reply:
if for example a contract needs more general resource access across all bank balances (because its send funds to a bunch of people), but only on a execute Msg used infrequently ("send_mass_funds" for example), we can use this message conditional with the format `".send_mass_funds"` so that it ONLY matches if thats the excute command being used, and in that case would invoke the widespread resource access.

TLDR, it mitigates widespread resource access use cases to the execute functions that use them.